Resources/public/templates/ezsearchsearchview.hbt

```diff
@@ -0,0 +1,14 @@
+<div class="pure-g">
+    <div class="ezsearch-searchview-content pure-u">
+        <header class="ez-page-header">
+            <h1 class="ez-page-header-name" data-icon="&#xe986;">Search</h1>
+        </header>
+        <div class="ezsearch-searchview-search-form">
+            <p class="ezsearch-searchview-search-form-title">Search content:<p>
```

Comment:
you should use a label related to its input for such text. Your markup would become something like (maybe with more classes):
```html
<form class="ezsearch-form pure-form">
   <p>
       <label for="searchstring">Search content:</label>
       <input id="searchstring" name="searchstring" type="text" value="{{inputString}}">
       <button type="submit" class="pure-button ez-button">Search</button>
   </p>
</form>